kolibri/core/auth/management/utils.py

```diff
@@ -22,6 +23,21 @@
 from kolibri.core.discovery.utils.network.errors import URLParseError
 
 
+class DisablePostDeleteSignal(object):
+    """
+    Helper that disables the post_delete signal temporarily when deleting, so Morango doesn't
+    create DeletedModels objects for what we're deleting
+    """
+
+    def __enter__(self):
+        self.receivers = post_delete.receivers
+        post_delete.receivers = []
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        post_delete.receivers = self.receivers
+        self.receivers = None
+
+
 def _interactive_client_facility_selection():
     facilities = Facility.objects.all().order_by("name")
     message = "Please choose a facility to sync:\n"
```

Comment:
@bjester Is the use of `sync` here intentional? This command should delete the facility, so why _sync_...? :thinking:

Reply:
@radinamatic Oh good catch! I didn't notice it said "sync". I'll make that dynamic

Reply:
Updated here https://github.com/learningequality/kolibri/pull/7187/commits/a41de91333175f594d4bf2d7ce1adaf0a7ed7bd5